virttest/qemu_devices/qcontainer.py

```diff
@@ -1438,6 +1438,12 @@ def machine_other(cmd=False):
                         cmd += ",accel=%s" % machine_accel
                     if machine_type_extra_params:
                         cmd += ",%s" % machine_type_extra_params.strip(',')
+                    machine_help = self.execute_qemu("-M %s,?" %machine_type)
```

Comment:
@huangyum does it work well with non-x86 platforms?

Reply:
Yes, test passed on ppc host.

Reply:
It works on arm and s390 platforms as well.

Reply:
Get it, thanks!